ddtrace/tracer/tracer.go

```diff
@@ -328,6 +329,7 @@ func (t *tracer) StartSpan(operationName string, options ...ddtrace.StartSpanOpt
 		// this is a brand new trace, sample it
 		t.sample(span)
 	}
+	log.Debug("Started Span: %v, Operation: %s, Resource: %s, Tags: %v, %v", span, span.Name, span.Resource, span.Meta, span.Metrics)
```

Comment:
Why do we need to log anything after `%v`? It should already include all of it. Seems redundant.